requirements-test.txt

```diff
@@ -1,4 +1,5 @@
-pytest>=3.0.0
-pytest-django
+pytest==3.3.2
```

Comment:
The 3 modified dependencies are declared in test-infra
We could remove them from this repo and cap it test-infra

Reply:
only "issue" is when one wants to run the test locally

Reply:
I don't think removing dependencies from the apps themselves is an option. I'd prefer to create a single meta-package that we can all depend on. Not sure on the naming yet, but perhaps `ome-pytest`.